### DISCUSSION TITLE
Can we run the model in hourly time steps?

### DISCUSSION THREAD

**Discussion by KaiganCodes** [iiasa/CWatM#111] [General]
Hi, I am Madhawa from Japan. I am interested in your model and would like to know whether we can use hourly time steps to run the model. Specially if we want to do event-based analysis, Running the model in hourly steps would be much more effective. Do you have any upcoming updates regarding this?

**Comment by dof1985** [comment]
Dear Madhawa,

Thanks for reaching out. CWatM's current version only supports a daily time step, though some modules run on sub-daily time steps (e.g., soil, lakes and reservoirs, and river routing). We have no specific plans to provide an hourly run option.

Best
Dor Fridman
